webapp/src/index.tsx

```diff
@@ -1,14 +1,15 @@
+import { createRoot } from 'react-dom/client';
 import React from 'react';
-import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
+import Landing from './pages/Landing';
```

Comment:
If you are importing `Landing `from `./pages/Landing`, what's the point of having an index.tsx file in the Landing folder importing and exporting it?

Reply:
Good point. I've added that file to not have to type Landing twice.
```
import Landing from './pages/Landing/Landing';\
```


Maybe adding a barrel file to each component it's a bit too much. We could add one barrel file inside the components directory to import them from one single file in one line.

```
import { Landing, SomePage, SomeOtherPage } from './pages';
```

What do we think about this?

Reply:
Ok, I got it. Yes, the idea of the barrel in the components folder sounds good. And one all the pages as well. I thought from that structure that there is going to be an index file for each page
/src
- /pages
--  /Landing
---   index.tsx 
---   Landing.tsx
---   Landing.css

Good 👍